Compile error on Raspberry Pi OS 32bit

## Description
```
pi@raspberrypi:~/go/src/gocv.io/x/gocv $ go run ./cmd/version/main.go 
# gocv.io/x/gocv
./core.go:2009:16: type [1073741824]*_Ctype_char larger than address space
./core.go:2009:16: type [1073741824]*_Ctype_char too large
```

The line goes like this:
```
tmpslice := (*[1 << 30]*C.char)(unsafe.Pointer(strs.strs))[:length:length]
```
And the fix is trivial,
```
tmpslice := (*[1 << 28]*C.char)(unsafe.Pointer(strs.strs))[:length:length]
```
As far as I can tell, it doesn't cause any problems, but I tested this in a limited capacity. Should I send a PR to address this?

## Your Environment

* Operating System and version:
Raspberry Pi OS
Version: August 2020
Release date: 2020-08-20
* OpenCV version used:
gocv version: 0.25.0
opencv lib version: 4.5.0
* How did you install OpenCV?
I ran the `make install` script, per website instructions.
* GoCV version used:
0.25.0
* Go version:
go version go1.15.3 linux/arm
* Did you run the `env.sh` or `env.cmd` script before trying to `go run` or `go build`?
No
* Hardware platform
Raspberry Pi 3

--- a/core.go
+++ b/core.go
@@ -2060,7 +2060,7 @@ func toGoBytes(b C.struct_ByteArray) []byte {
 // Converts CStrings to a slice of Go strings even when the C strings are not contiguous in memory
 func toGoStrings(strs C.CStrings) []string {
 	length := int(strs.length)
-	tmpslice := (*[1 << 30]*C.char)(unsafe.Pointer(strs.strs))[:length:length]
+	tmpslice := (*[1 << 20]*C.char)(unsafe.Pointer(strs.strs))[:length:length]
 	gostrings := make([]string, length)
 	for i, s := range tmpslice {
 		gostrings[i] = C.GoString(s)

